tagreader.yaml

```diff
@@ -1,22 +1,26 @@
 # Insert your SSID and Your PWD after inital setup
 wifi:
-  networks:
+# networks:
 #    - ssid: 'REPLACEME'          # Enter your WiFi SSID here. Example: `ssid: 'your_network_name'`
 #      password: 'REPLACEME'      # Enter your wifi password here. Example: `password: 'abcde123456'`
   ap:
-    ssid: ${devicename}
+    ssid: ${name}
 
 # Enable the captive portal for inital WiFi setup
 captive_portal:
 
+dashboard_import:
+  package_import_url: github://adonno/tagreader/tagreader.yaml
```

Comment:
This needs to have a tag. If you don't add a tag, it will always update from the default branch (`master` in your case). This means that the default branch should **always** point at this hardware revision that this firmware ships on. If you were to move pins around, everyone would receive an update breaking their tagreader.

So use tags

Reply:
What if on a hardware change I create a new yaml? And support both ? 

Like having esp32 doesn't exclude working with esp8266? I would create a tag_reader_32.yaml ?

Reply:
yeah that will work.